### ISSUE TITLE
It is recommended to add a function of linking data with objects in the map 

### ISSUE THREAD

**Issue by anders86755** [RE0x2A/geojson.io-for-vscode#28]
Hey rend, 

Could you consider adding a feature? The function is probably like this:

Place a toggle button on the map. When entering a strong association state, click an object on the map to automatically scroll the data window to the data corresponding to the object. When you click on data in the data window, the map automatically focuses on the feature corresponding to the data. When this button is clicked again, it returns to its normal state. 

I think this should be a very useful feature.

**Comment by 42arch** [comment]
Yes, this sounds cool and very useful, but there are a few technical issues that need to be solved to achieve this feature. I will consider your suggestion and try to find an easy way. If this feature is achievable, I will add it.
